{"name": "Нельзя создать пользователя c неверным полем email", "status": "passed", "start": 1718864256841, "stop": 1718864257221, "uuid": "b3a2d93f-3e7b-4ffc-98f1-d148e996bc47", "historyId": "bdea1ed30a18d95de63dbef3b59241da", "testCaseId": "bdea1ed30a18d95de63dbef3b59241da", "fullName": "tests.test_create_user.TestCreateUser#test_create_user_with_incorrect_field_email_code_403", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "16004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}